db/migrate/20160512012637_add_thumbnail_to_serials.rb

```diff
@@ -0,0 +1,5 @@
+class AddThumbnailToSerials < ActiveRecord::Migration
+  def change
+    add_column :serials, :thumbnail, :string, null: false
```

Comment:
When `serials.thumbnail` is already registered as `NULL`, this migration causes the following error:
```
PG::NotNullViolation: ERROR:  column "thumbnail" contains null values
```